Source/Header/SwipeLabelView.swift

```diff
@@ -7,7 +7,13 @@ public final class SwipeLabelView: UIView, DayViewStateUpdating {
     case Backward
   }
 
-  public private(set) var calendar = Calendar.autoupdatingCurrent
+  public var calendar = Calendar.autoupdatingCurrent {
+    didSet {
+      // I doubt that someone will use this view without a state, but
```

Comment:
It's good to keep CK modular, as there are already use-cases when a few CK modules are used on their own or configured by the developer in a non-standard way. So, it's a great idea to keep the modularity going.